### ISSUE TITLE
`kleborate --version` reports v2.2.0, expect 2.3.1

### ISSUE THREAD

**Issue by kapsakcj** [katholt/Kleborate#68]
Hi, I've installed kleborate following directions from here: https://github.com/katholt/Kleborate/wiki/Installation

And found that `kleborate --version` is reporting `Kleborate v2.2.0` when I expect it to report 2.3.1

I think it may be because the `version.py` file included in the v2.3.1 github tag specifies `2.2.0`? https://github.com/katholt/Kleborate/blob/v2.3.1/kleborate/version.py

If this is true could you please update this file and re-release 2.3.1 or perhaps release 2.3.2?

Curtis

**Comment by singuyenmai** [comment]
I was confused by this version mismatch, too. 

It seems like the same issue also happens with Kaptive. I pulled both the Kleborate and Kaptive repos just today. Kaptive version should be v2.0.5, but running the script returned v2.0.3. It turns out that the` __version__` variable in the `kaptive.py` script was not updated accordingly.

Checked the commits logs of both repos. Lastest commits were corresponding to the latest versions.
